frontend/src/features/intent-draft-read/ui/IntentTreePanel.module.css

```
@@ -0,0 +1,188 @@
+  gap: 8px;
+}
+
+.depthGuide {
+  position: absolute;
+  left: 0;
+  top: 6px;
+  bottom: 6px;
+  width: 2px;
+  background-image: repeating-linear-gradient(
+    to bottom,
+    currentColor 0 4px,
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)